aether/sdk/conf/storages.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2020 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+from storages.backends.s3boto3 import S3Boto3Storage
+from storages.backends.gcloud import GoogleCloudStorage
+
+
+def get_required(name):
+    try:
+        return os.environ[name]
+    except KeyError as key:
+        raise RuntimeError(f'Missing {key} environment variable!')
+
+
+class StaticS3(S3Boto3Storage):
+    bucket_name = get_required('STATIC_BUCKET_NAME')
```

Comment:
Why not `bucket_name = settings.STATIC_BUCKET_NAME`?

Reply:
because I am not setting `STATIC_BUCKET_NAME` on settings

Reply:
See my comment above :stuck_out_tongue:

Reply:
I mean, in the settings file